script.tvmaze.scrobbler/libs/tvmaze_api.py

```diff
@@ -52,7 +52,8 @@ class ApiError(Exception):
     @staticmethod
     def extract_error_message_from_response(response):
         # type: (requests.Response) -> Text
-        if 'application/json' in response.headers.get('Content-Type', ''):
+        if 'application/json' in response.headers.get('Content-Type', '') \
+                and len(response.content) > 0:
```

Comment:
In Python you don't need to check the length of a collection object. A non-empty collection evaluates to `True` by itself.